generatebundlefile/hack/release_prod.sh

```diff
@@ -85,12 +85,12 @@ fi
 
 export AWS_PROFILE=prod
 export AWS_CONFIG_FILE=${BASE_DIRECTORY}/generatebundlefile/prodconfigfile
-for version in 1-22 1-23 1-24 1-25 1-26 1-27; do
+for version in 1-22 1-23 1-24 1-25 1-26 1-27 1-28; do
```

Comment:
Maybe in the future this should just iterate over everything in those directories.